src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -1,30 +1,260 @@
-# genai-perf
+<!--
+Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 
-## Installation
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+ * Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+ * Neither the name of NVIDIA CORPORATION nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
 
-### Install from Source
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# GenAI-Perf
+
+A tool to facilitate benchmarking generative AI models leveraging NVIDIA’s
+[performance analyzer tool](https://github.com/triton-inference-server/client/tree/main/src/c%2B%2B/perf_analyzer).
+
+GenAI-Perf builds upon the performant stimulus generation of the performance
+analyzer to easily benchmark LLMs. Multiple endpoints are currently supported.
+
+The GenAI-Perf workflow enables a user to
+* [Generate prompts](#model-inputs) using either
+  * synthetic generated data
+  * open orca or CNN daily mail datasets
+* Transform the prompts to a format understood by the
+[chosen endpoint](#basic-usage)
+  * Triton Infer
+  * OpenAI
+* Use Performance Analyzer to drive stimulus
+* Gather LLM relevant [metrics](#metrics)
+* Generate reports
+
+all from the [command line](#cli).
+
+Note: GenAI-Perf is currently in early release while under rapid development.
+While we will try to remain consistent, command line options are subject to
+change until the software hits 1.0. Known issues will also be documented as the
+tool matures.
```

Comment:
```suggestion
> [!Warning]
> GenAI-Perf is currently in early release while under rapid development.
> While we will try to remain consistent, command line options are subject to
> change until the software hits 1.0. Known issues will also be documented as the
> tool matures.
```

Reply:
[!Note] is probably sufficient instead of [!Warning]

Reply:
Done